src/dnaio/_core.pyx

```diff
@@ -148,6 +148,24 @@ cdef class Sequence:
         """
         return self.fastq_bytes(two_headers=True)
 
+    def is_mate(self, Sequence other):
```

Comment:
This should get a docstring considering we want to make API documentation available soon.